2. Services/2.2 Processings/2.2 Processings.md

```diff
@@ -2,9 +2,9 @@
 
 ## 2.2.0 Introduction
 
-Processing services are the layer where a higher order of business logic is implemented. They may combine (or orchestrate) two primitive-level functions from their corresponding foundation service to introduce a newer functionality. They may also call one primitive function and change the outcome with a little bit of added business logic. And sometimes processing services are there as a pass-through to introduce balance to the overall architecture.
+A Processing service is a layer that implements a higher order of business logic. It may combine (or orchestrate) two primitive-level functions from their corresponding foundation service to introduce a newer functionality. They may also call one primitive function and change the outcome with a little bit of added business logic. And sometimes, processing services are there as a pass-through to introduce balance to the overall architecture.
 
-Processing services are optional, depending on your business need - in a simple CRUD operations API, processing services and all the other categories of services beyond that point will cease to exist as there is no need for a higher order of business logic at that point.
```

Comment:
Why the `all other categories of services` was removed?

Reply:
I condensed to 

> processing services will cease to exist _**beyond that point**_

which means conveys that same meaning with fewer words.